src/brokers/add-broker/components/AddBrokerForm/FormPage.component.tsx

```diff
@@ -1,38 +1,81 @@
 import { FC, useState } from 'react';
-import { Form, FormGroup, TextInput, Alert } from '@patternfly/react-core';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  Alert,
+  Button,
+  AlertGroup,
+  Grid,
+  GridItem,
+  ActionListItem,
+  ActionList,
+} from '@patternfly/react-core';
 import { useTranslation } from '../../../../i18n';
+import { Link } from 'react-router-dom';
+type validate = 'success' | 'warning' | 'error' | 'default';
 
 export const FormPage: FC = () => {
```

Comment:
File and component name  `FormView` would be better.

Reply:
okay Ajay i will change the name.